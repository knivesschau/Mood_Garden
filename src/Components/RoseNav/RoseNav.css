.Rose_Nav {
    display: flex;
    flex-direction: row;
    margin-left: 1em;
}