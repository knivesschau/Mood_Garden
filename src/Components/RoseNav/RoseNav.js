import React, {Component} from 'react';
import {Link} from 'react-router-dom';
import TokenService from '../../services/token-services';
import './RoseNav.css'

export default class RoseNav extends Component {
    render() {
        return (
            <div className="Rose_Nav">
                <Link to="/your-garden">
                    Back To Your Garden
                </Link>
            </div>
        )
    }
}