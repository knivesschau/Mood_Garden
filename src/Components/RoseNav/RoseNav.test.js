import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import RoseNav from './RoseNav';

describe ('RoseNav Component', () => {
    it ('renders the complete RoseNav Component', () => {
        const wrapper = shallow(<RoseNav/>);
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})