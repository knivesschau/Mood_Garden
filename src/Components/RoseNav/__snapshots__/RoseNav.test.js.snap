// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoseNav Component renders the complete RoseNav Component 1`] = `
<div
  className="Rose_Nav"
>
  <Link
    to="/your-garden"
  >
    Back To Your Garden
  </Link>
</div>
`;
